### ISSUE TITLE
Rollup failed to resolve import "#components" in 2.13.3

### ISSUE THREAD

**Issue by dolbex** [nuxt/content#2812]
### Environment

------------------------------
- Operating System: Darwin
- Node Version:     v18.18.2
- Nuxt Version:     3.13.2
- CLI Version:      3.13.2
- Nitro Version:    2.9.7
- Package Manager:  bun@1.1.29
- Builder:          -
- User Config:      compatibilityDate, devtools, modules, extends, runtimeConfig, typescript, nitro, app, alias, routeRules, basicAuth, content, csurf, socialShare, voix, cron, css, image, postcss, s3, vite, serverMiddleware
- Runtime Modules:  nuxt-cron@^1.7.0, nuxt-icon@^0.6.10, @nuxt/image@1.8.1, @vueuse/nuxt@11.1.0, @pinia/nuxt@0.5.5, @logicbomb-media/voix@2.36.1, @stefanobartoletti/nuxt-social-share@^1.1.2, @bg-dev/nuxt-s3@^2.0.1, @kgierke/nuxt-basic-auth@^1.6.0, nuxt-csurf@^1.6.3
- Build Modules:    -
------------------------------

### Reproduction

We experienced this during build both locally and on Github actions. I'm not quite sure how to reproduce a build error experience.

### Describe the bug

Update from 2.13.2 to 2.13.3 resulted in this error for us:

`#13 19.06  ERROR  Nuxt Build Error: [vite]: Rollup failed to resolve import "#components" from "virtual:nuxt:/home/bun/app/.nuxt/mdc-image-component.mjs".`

### Additional context

Only content I believe we have in this particular project is a json file with an array of objects like

```
[
  {
    "id": 1,
    "name": "Goroka",
    "city": "Goroka",
    "country": "Papua New Guinea",
    "faa_code": "GKA",
    "icao_code": "AYGA",
    "lat": -6.081689,
    "lng": 99.99999999,
    "altatude": 5282,
    "timezone": 10,
    "dst": "U",
    "timezone_wiki_uri": "Pacific/Port_Moresby"
  },
]
```

**Comment by dolbex** [comment]
Looks like this might be resolved with the update in 2.13.4 with the bump to mdc. Testing.

**Comment by farnabaz** [comment]
Thanks for the report. Should be fixed in `2.13.4` hotfix.

Checkout #2811

**Comment by dolbex** [comment]
Thanks for the quick response @farnabaz !
